### PR TITLE
Add new bundles for each CXF jar that we pull in for the JAXWS-2.3 feature.

### PR DESCRIPTION
**Pull request by WhiteCat22**
Add new bundles for each CXF jar that we pull in for the JAXWS-2.3 feature.

### CODE REVIEW

**Comment by WhiteCat22** [comment]
#6221

**Comment by WhiteCat22** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides [hunk 1]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.bindings.soap.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides]: This is probably nit-picking, but I was planning on matching the projects, bundle, and packages names what the jaxrs projects does between versions of the jaxrs feature I.E. com.ibm.ws.jaxws.2.3.common etc etc. Would you mind changing the packages to match that? So com.ibm.ws.jaxws.2.3.cxf.rt.binding.soap.NoOpActivator

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2/bnd.overrides [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.bindings.xml.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2/bnd.overrides]: Same package name concern as above comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2/src/com/ibm/ws/jaxws23/cxf/rt/bindings/xml/NoOpActivator.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.bindings.xml;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.xml.3.2/src/com/ibm/ws/jaxws23/cxf/rt/bindings/xml/NoOpActivator.java]: Same package name concern as the first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/bnd.bnd [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/bnd.bnd]: Copyright year to 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/bnd.overrides [hunk 5]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.databinding.jaxb.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/com/ibm/ws/jaxws23/cxf/rt/databinding/jaxb/NoOpActivator.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.databinding.jaxb;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/com/ibm/ws/jaxws23/cxf/rt/databinding/jaxb/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.bnd [hunk 7]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.bnd]: Update copyright year to 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.overrides [hunk 8]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.frontend.jaxws.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/jaxws/NoOpActivator.java [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.frontend.jaxws;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/jaxws/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/jaxws/package-info.java [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.rt.frontend.jaxws;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/jaxws/package-info.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.bnd [hunk 11]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.bnd]: Update copyright to 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.overrides [hunk 12]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.frontend.simple.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/simple/NoOpActivator.java [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.frontend.simple;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/simple/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/simple/package-info.java [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.rt.frontend.simple;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/src/com/ibm/ws/jaxws23/cxf/rt/frontend/simple/package-info.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.bnd [hunk 15]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.bnd]: Update copyright year to 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.overrides [hunk 16]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.ws.addr.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/addr/NoOpActivator.java [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.ws.addr;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/addr/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/addr/package-info.java [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.rt.ws.addr;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/addr/package-info.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.bnd [hunk 19]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.bnd]: Update copyright year 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.overrides [hunk 20]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.ws.policy.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/policy/NoOpActivator.java [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.ws.policy;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/policy/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/policy/package-info.java [hunk 22]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.rt.ws.policy;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/com/ibm/ws/jaxws23/cxf/rt/ws/policy/package-info.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.bnd [hunk 23]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.bnd]: Update copy right year to 2019

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.overrides [hunk 24]
```diff
@@ -0,0 +1,14 @@
+-include= ~../cnf/resources/bnd/rejar.props
+bVersion=1.0
+cxfVersion=3.2.6
+
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2
+
+Bundle-Activator: com.ibm.ws.jaxws23.cxf.rt.wsdl.NoOpActivator
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.overrides]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/src/com/ibm/ws/jaxws23/cxf/rt/wsdl/NoOpActivator.java [hunk 25]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.wsdl;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/src/com/ibm/ws/jaxws23/cxf/rt/wsdl/NoOpActivator.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/src/com/ibm/ws/jaxws23/cxf/rt/wsdl/package-info.java [hunk 26]
```diff
@@ -0,0 +1,12 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+@org.osgi.annotation.versioning.Version("1.0.0")
+package com.ibm.ws.jaxws23.cxf.rt.wsdl;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/src/com/ibm/ws/jaxws23/cxf/rt/wsdl/package-info.java]: Same package name concern as first comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/com/ibm/ws/jaxws23/cxf/rt/bindings/soap/NoOpActivator.java [hunk 27]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws23.cxf.rt.bindings.soap;
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/com/ibm/ws/jaxws23/cxf/rt/bindings/soap/NoOpActivator.java]: Same package name concern as above comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides [hunk 28]
```diff
@@ -0,0 +1,14 @@
...
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+Import-Package: \
+
+Export-Package: \
+  com.ibm.ws.jaxws23.cxf.rt.bindings.soap,\
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides]: Same package name concern as above comment

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides (hunk 1 above)
Comment (WhiteCat22) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.overrides]: eclipse will not allow a number by itself in a package name

**Pull request merged**